frontend/app/professor/simulation-builder/page.tsx

```
@@ -1983,258 +2234,273 @@ return (
+         {/* Tab Navigation */}
+         <div className="flex border-b border-gray-200 mb-6">
+           <button
+             onClick={() => setActiveTab('configuration')}
+             className={`flex items-center gap-2 px-6 py-3 font-medium text-sm transition-colors ${
+               activeTab === 'configuration'
+                 ? 'border-b-2 border-black text-black'
+                 : 'text-gray-600 hover:text-gray-900'
+             }`}
+           >
+             <Settings className="h-4 w-4" />
+             Configuration
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)